api/v1alpha1/flowcollector_webhook.go

```diff
@@ -59,6 +59,10 @@ func (r *FlowCollector) ConvertTo(dstRaw conversion.Hub) error {
 
 	dst.Spec.Loki.StatusTLS = restored.Spec.Loki.StatusTLS
 
+	if restored.Spec.Exporters != nil {
+		dst.Spec.Exporters = restored.Spec.Exporters
+	}
```

Comment:
Should we merge both existing + restored has these can be different instead of erease existing with restored ?

Reply:
Done!